torch/_inductor/config.py

```diff
@@ -997,6 +997,12 @@ class cpp:
     # decomposed into 7x4x2 thread blocks along MxNxK of a GEMM.
     gemm_thread_factors = os.environ.get("TORCHINDUCTOR_CPP_GEMM_THREAD_FACTORS", None)
 
+    # Set GEMM Transverse strategy: support VERTICAL, HORIZONTAL
+    # If both are enabled, the strategy will be selected based on the heuristic.
+    cpp_gemm_transverse_strategy = os.environ.get(
```

Comment:
nit: you don't have to add "cpp_" prefix to the configuration name since it is already inside the `cpp` class.